ingest-app/src/migrations/010_add_unique_constraint_bulk_update_status.clj

```diff
@@ -0,0 +1,21 @@
+(ns migrations.010-add-unique-constraint-bulk-update-status
+  (:require
+   [clojure.java.jdbc :as j]
+   [config.migrate-config :as config]))
+
+(defn up
+  "Migrates the database up to version 10."
+  []
+  (println "migrations.010-add-unique-constraint-bulk-update-status up...")
+  (j/db-do-commands (config/db)
```

Comment:
This will fail if there is any existing data in the system which violates the constraint. Have you verified the names are unique in all environments?

I think it's fine to leave the migration like this, but we should try to address any data issues just prior to deployment.